packages/discord.js/src/managers/ApplicationCommandManager.js

```diff
@@ -232,6 +232,7 @@ class ApplicationCommandManager extends CachedManager {
       type: command.type,
       options: command.options?.map(o => ApplicationCommand.transformOption(o)),
       default_permission: command.defaultPermission ?? command.default_permission,
+      default_member_permissions: command.default_member_permissions ?? command.defaultMemberPermissions,
```

Comment:
We check for camelCase properties first, and fallback to snake_case:
```suggestion
      default_member_permissions: command.defaultMemberPermissions ?? command.default_member_permissions,
```